schema.yaml

```diff
@@ -286,7 +284,7 @@ components:
             responses:
               type: object
               required:
-                - 200
+                - "200"
```

Comment:
@pgeadobe can you elaborate on what's prompting this change?  json schema definitely allows naked numbers in this context and this may break swagger compatibility, as all of our response codes are unquoted.

Reply:
Without this change, swaggest json-schema would complain "string expected" for array element(s) of "required" property.
Presumably the same thing Ted was commenting in slack channel: 
"required
The value of this keyword MUST be an array. Elements of this array, if any, MUST be strings, and MUST be unique."